Integration template: add includes (#45336)
home-assistant/home-assistant.github.io: source/_integrations/_integration_docs_template.markdown

  - **Available for machines**: all

<!--

> The "includes" below add sections (heading 2) for triggers, conditions, or actions for this integration.

Author: ```suggestion
The "includes" elements below add sections (heading 2) for triggers, conditions, or actions for this integration. Uee the one that applies to what you are adding.
```

> The "includes" elements below add sections (heading 2) for triggers, conditions, or actions for this integration. Uee the one that applies to what you are adding.

Author: ```suggestion
The "include" elements below add sections (heading 2) for triggers, conditions, or actions for this integration. Use the one that applies to what you are adding.
```

Create separate files for each:
- trigger, in source/_triggers/
- condition, in source/_conditions/
- action, in source/_actions/
-->
{% include integrations/actions.md %}
{% include integrations/conditions.md %}

> {% include integrations/triggers.md %}

Reviewer: Should we write them like this instead?
```suggestion
{% include integrations/actions.md %}

{% include integrations/conditions.md %}

{% include integrations/triggers.md %}
```

<!--- If the integration has triggers, conditions, and actions, use the combined include. -->

> 

Reviewer: Thank you you for updating the template @c0ffeeca7! 😊
This is a good point. Maybe we can leave the combined include as the default and put the single includes in the comment, for example.

> 

Author: I disagree. Many integrations only add actions. I think that is what we see for the majority of contributed integrations.

> 

Author: The Lab features trigger/conditions/action documentation we are currently adding are for internal integrations. They are not the typical use case contributors will encounter.

> <!--- If the integration has all three components (triggers, conditions, and actions), use the combined include. -->

Author: ```suggestion
<!--- If the integration has all three components (triggers, conditions, and actions), use the combined include instead. -->
```

> {% include integrations/triggers_conditions_actions.md %}

Reviewer: Agreed. Then we can comment the combined include instead:

```suggestion
<!--
If the integration has all three components (triggers, conditions, and actions), use the combined include instead. 
{% include integrations/triggers_conditions_actions.md %}
-->
```
